src/main/java/ua/softserve/rv_028/issuecitymonitor/controller/RegistrationController.java

```diff
@@ -23,8 +27,10 @@ public Boolean checkEmail(@RequestParam("email") String email){
     @PostMapping(path = "/api/registration")
     public HttpStatus userRegistration(@RequestBody UserDto user) {
```

Comment:
You should move logic to service, throw exceptions from service and catch them from advicecontroller. Also this must return void.